src/helpers.ts

```diff
@@ -27,18 +27,17 @@ export function readSettings<T>(serviceName: string, settingsFileName: string):
     throw new Error(`[${serviceName}] Unable to load configuration file ${settingsFileName}.`);
   }
 
-  let parseErrors: JSONC.ParseError[];
+  const parseErrors: JSONC.ParseError[] = [];
```

Comment:
This was needed because of https://github.com/danecreekphotography/node-deepstackai-trigger/pull/378/files#r502622394

I wanted to leave this alone, as I'm not sure what will happen at run time of the actual application.